GoogleDataTransport/GDTCORLibrary/Public/GDTCORClock.h

```diff
@@ -27,10 +27,10 @@ NS_ASSUME_NONNULL_BEGIN
 /** The offset from UTC in seconds. */
 @property(nonatomic, readonly) int64_t timezoneOffsetSeconds;
 
-/** The kernel boot time when this clock was created. */
+/** The kernel boot time when this clock was created in nanoseconds. */
 @property(nonatomic, readonly) int64_t kernelBootTime;
 
-/** The device uptime when this clock was created. */
+/** The device uptime when this clock was created in nanoseconds. */
 @property(nonatomic, readonly) int64_t uptime;
```

Comment:
You didn't want to rename the property?

Reply:
I'm a bit hesitant doing this because it will be another breaking change. Though, I may deprecate it and create another with units in the name.

Reply:
@paulb777 @ryanwilson Any objections against a new property + deprecation?

Reply:
Fine with me - as long as we avoid a breaking change.